lib/service.js

```diff
@@ -74,4 +74,24 @@ export function createService(resolver, token, shutdown) {
         process.stderr.write = stderr_write;
       });
   };
+
+  /**
+   * @param {Socket} con
+   * @param {string} command
+   */
+  function onCommand(con, command) {
+    switch (command) {
+      case 'ESLINT_D_STOP':
+        shutdown();
+        con.end();
+        return;
```

Comment:
Nitpick: You could use `break` here and move `con.end()` to the end of the block. It's called in each of the cases.

Reply:
fixed